packages/pds/src/app-view/services/indexing/plugins/follow.ts

```diff
@@ -49,15 +48,17 @@ const findDuplicate = async (
   return found ? new AtUri(found.uri) : null
 }
 
-const eventsForInsert = (obj: IndexedFollow) => {
+const notifsForInsert = (obj: IndexedFollow) => {
   return [
-    messages.createNotification({
+    {
       userDid: obj.subjectDid,
       author: obj.creator,
       recordUri: obj.uri,
       recordCid: obj.cid,
-      reason: 'follow',
-    }),
+      reason: 'follow' as const,
```

Comment:
While it is on my mind, I think I saw a user report of muting somebody, but still getting notifications if they follow/unfollow repeatedly.